### PR TITLE
OSDOCS-3889: release note for NE-942

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
This is the release note for https://issues.redhat.com/browse/OSDOCS-3889
Engineering story: https://issues.redhat.com/browse/NE-942: Add field in installer to get LB type for AWS

Version(s):
PR applies to : 4.12 only

Link to docs preview: 
https://51186--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-aws-load-balancer-customization

Additional information:
WIP docs content here: https://github.com/openshift/openshift-docs/pull/51460
PR replaces 51174

QE review:
- [Y] QE has approved this change.

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@miheer and @lihongan 
Please take a look and either approve or suggest revisions. Thank you!

**Comment by ShaunaDiaz** [comment]
@bmcelvee If you'd like to take a look for the ROSA side.

**Comment by lihongan** [comment]
/lgtm
Thank you!

**Review by miheer — CHANGES_REQUESTED** [review]

**Review by miheer — CHANGES_REQUESTED** [review]

**Comment by ShaunaDiaz** [comment]
@miheer @Miciah Let's go back to the shorter RN text and I will footnote the table in the [docs](https://github.com/openshift/openshift-docs/pull/51460) entry.

**Comment by ShaunaDiaz** [comment]
@miheer @Miciah PTAL and let me know if this PR looks good now.

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -47,6 +47,12 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-12-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-12-aws-load-balancer-customization"]
+==== Specify the load balancer type in AWS during installation
+Beginning with {product-title} {product-version}, you can specify either NLB or Classic as a persistent load balancer type in AWS during installation. Afterwards, if an ingress controller is deleted, the load balancer type persists with the lbType configured during installation.
```
Comment (jboxman-rh) [inline release_notes/ocp-4-12-release-notes.adoc:52]: Do we know what NLB is? I probably should known this, but I don't.

We're capitalizing Ingress Controller, although I don't recall why: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#ingress-controller

This seems related to: https://kubernetes.io/docs/concepts/services-networking/service/#aws-nlb-support

Is `lbType` an object field?

**Review by jboxman-rh — COMMENTED** [review]
Not sure if it makes sense to clarify this a bit further. I'm not that familiar with AWS particulars.

**Comment by jboxman-rh** [comment]
Looks like this describes the particulars: https://github.com/openshift/api/pull/1209/files

But that just tells it to create the Ingress Controller with the right LB type. So that API field is in the Ingress API somewhere.

**Pull request merged**